### DISCUSSION TITLE
Install issue

### DISCUSSION THREAD

**Discussion by prewebdesign** [wintercms/winter#971] [Q&A]
Fresh install of v1.2.3 - all green on install wizard. Error message on "Determining dependencies..."

**Unable to determine dependencies for Winter CMS. Class "Composer\Filter\PlatformRequirementFilter\IgnoreNothingPlatformRequirementFilter" not found**

I've tried running the installer on PHP8, 8.1 and 8.2 and I get the same issue on each. It worked perfectly on another client on the same server so not sure why I am seeing this? Any words of wisdom from anyone please?

**Comment by bennothommo** [comment]
There should be an install.log file in the folder you tried to install in. Could you provide that to us?
